low-level/framework/base/sspl_constants.py

```diff
@@ -86,26 +86,6 @@
 # TODO Keep only constants in this file.
 # other values(configs) should come from config.
 
-# TODO: Below finding machine id/node_key_id code will be replaced by PR #281 [EO-16515]
-try:
-    with open("/etc/machine-id") as f:
-        node_key_id = f.read().strip("\n")
-except Exception as err:
-    print("Failed to get machine-id. - %s" % (err))
-
-storage_type = Conf.get('global_config',
```

Comment:
these all are directly consumed from ConfStor now?

Reply:
Yes. In `low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable.py`